scripts/build-catalogue.in

```diff
@@ -108,8 +108,12 @@ with TemporaryDirectory() as tmp:
         fd.write(cmake)
     shutil.copy2(f'@ARB_INSTALL_DATADIR@/BuildModules.cmake', tmp)
     shutil.copy2(f'@ARB_INSTALL_DATADIR@/generate_catalogue', tmp)
-    sp.run('cmake ..', shell=True, check=True, capture_output=not verbose)
-    sp.run('make',     shell=True, check=True, capture_output=not verbose)
+    if verbose:
+        sp.run('cmake ..', shell=True, check=True)
+        sp.run('make',     shell=True, check=True)
+    else:
+        p = sp.run('cmake ..', shell=True, check=True, stdout=sp.DEVNULL, stderr=sp.DEVNULL)
+        p = sp.run('make',     shell=True, check=True, stdout=sp.DEVNULL, stderr=sp.DEVNULL)
```

Comment:
Ok, that works and is even shorter. Two suggestions 
- do not capture `p`, since we never use it
- currently, we would duplicate the whole call when actually just want to change stdout/err, so how about this?
   ```py
   out, err = (None, None) if verbose else (sp.DEVNULL, sp.DEVNULL)
   sp.run('make',     shell=True, check=True, stdout=out, stderr=err)
   # ...
   ```
Reduce the duplication, thus easing maintenance, but trades for the dreaded ternary op.

Reply:
```python
if verbose:
  out, err = (sp.PIPE, sp.PIPE)
else:
  out, err = (None, None)
```

Reply:
I use @Helveg proposition, but explicitly pipe to `DEVNULL` in the nonverbose case.